components/pagination/nz-pagination.component.html

```diff
@@ -26,7 +26,7 @@
                [value]="nzPageIndex"
                (keydown.enter)="handleKeyDown($event,simplePagerInput,false)"
                size="3">
-        <span class="ant-pagination-slash">／</span>
```

Comment:
Thank you for your PR, but fullwidth is expected.
https://github.com/react-component/pagination/blob/master/src/Pagination.jsx#L429

@afc163 Any suggestions?

Reply:
We should fix too.

Reply:
https://github.com/react-component/pagination/pull/220